tools/topology/topology1/sof-tgl-max98373-rt5682.m4

```diff
@@ -91,13 +91,16 @@ define(KWD_PIPE_SCH_DEADLINE_US, 20000)
 # include the generic dmic with kwd
 include(`platform/intel/intel-generic-dmic-kwd.m4')
 
+ifdef(`INCLUDE_IIR_EQ',`',
```

Comment:
This should be BT specific rather than IIR specific check (as there are IIR uses outside of BT)

Reply:
Let me investigate a little further, there should be a more convincing cause of the failure.

Reply:
Thanks @ranj063 for sharing https://github.com/thesofproject/linux/pull/3005, with that we can confirm this is indeed a resource related issue:

```
# code block
      755913.147046] (          18.229166) c0 ssp-dai      1.0   /drivers/intel/ssp/ssp.c:603  ssp_set_config(), ssrsa = 0x00000003, sstsa = 0x00000003
[      755932.469962] (          19.322916) c0 dai                           src/lib/dai.c:188  dai_put type 1 index 0 new sref 2
[      755958.772044] (          26.302082) c0 dai          2.13           src/audio/dai.c:706  dai_config() dai type = 1 index = 0 dd 0x9e142100
[      755982.886627] (          24.114582) c0 dai          2.13           src/audio/dai.c:736  ERROR dai_config(): No memory for dai_config.
[      756001.115793] (          18.229166) c0 ipc                   src/ipc/helper-ipc3.c:487  ERROR ipc_comp_dai_config(): comp_dai_config() failed
```